extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -237,6 +240,17 @@
     @ConfigItem
     public boolean enableDecompression;
 
+    /**
+     * List of media types for which the compression should be enabled automatically, unless declared explicitly via {@link Compressed} or {@link Uncompressed}. 
+     */
+    @ConfigItem(defaultValue = "text/html,text/plain,text/xml,text/css,text/javascript,application/javascript")
+    public Optional<List<String>> compressMediaTypes;
```

Comment:
Are this list also used when you return a response from resteasy reactive?

Reply:
Yes, it should be. Although it's not tested yet. It's however not used for resteasy classic.

Reply:
I wonder if we should honor this config property for reasteasy-classic or ignore it completely? Currently, reasteasy-classic has a separate [config and annotation](https://quarkus.io/guides/rest-json#gzip-support).

Reply:
For now this config property and `enableCompression` is just ignored, i.e. it has no effect on http compression of resteasy classic endpoints.